dev-util/colm/colm-0.13.0.5-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="COmputer Language Manipulation"
+HOMEPAGE="https://www.colm.net/open-source/colm/"
+SRC_URI="https://www.colm.net/files/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
```

Comment:
License is wrong. Please recheck, as homepage and source code state different.

Reply:
I'll fix this in another PR

Reply:
I don't think, that this will be accepted this way.